migrations/2_deploy_contracts.js

```diff
@@ -3,7 +3,7 @@ var GameJam = artifacts.require("./GameJam.sol");
 module.exports = async (deployer, network) => {
   console.log(`deploying on ${network} network`)
 
-  deployer.deploy(GameJam);
+  await deployer.deploy.apply(deployer, [GameJam, 42])
```

Comment:
Is there a better way I can do this ?
I could do something like 

```suggestion
  const initialBalance = 42
  await deployer.deploy.apply(deployer, [GameJam, initialBalance])
```
But is there an even better way ?

Reply:
Please do it this way, we want to avoid `magic numbers` :)

Reply:
Not sure if there's a better way than this, maybe @CaelRowley can learn us 😄

Reply:
Changed this to the suggested change and now setting it to zero initially. We may need to change the setting to zero for testing purposes, what do you think?

Reply:
Tbh it's perfect for the moment! gg